### DISCUSSION TITLE
MET on Casper

### DISCUSSION THREAD

**Discussion by CPKalb** [dtcenter/METplus#2531] [Existing Builds] [answered]
Hi:

I am trying to run Grid-Stat using METplus on casper.  However, when I run, I'm getting a MET error:
/glade/work/dtcrt/METplus/derecho/components/MET/installations/11.1.0/bin/grid_stat: error while loading shared libraries: libpmi.so.0: cannot open shared object file: No such file or directory

I loaded MET and METplus using the commands from the existing builds page:
module use /glade/work/dtcrt/METplus/casper/components/MET/installations/modulefiles
module load met/11.1.0
module use /glade/work/dtcrt/METplus/casper/components/METplus/installations/modulefiles
module load metplus/5.1.0

It looks like a library error.  Any ideas what I can do to resolve the error?

Thanks,
Tina

**Comment by jprestop** [comment]
Hi @CPKalb.  Can you please send the command you are using so that I can try to recreate the problem you're experiencing?  When I simply run `grid_stat`, I do get the usage statement and not the error.

**Comment by CPKalb** [comment]
I am using the following command, running from /glade/u/home/kalb/ConvWx
run_metplus.py GridStat_fcstHRRR_obsNSSL_obj_ctest.conf system.conf

I think you should be able to see/read this directory, but if not let me know.  You will probably have to point OUTPUT_BASE in system.conf to one of your directories.  The error always appears in the output log file.

Tina

**Reply by CPKalb** [reply, marked as answer]
Wait, I found the error.  Although I loaded the correct modules for casper, my system configuration file had MET_INSTALL_DIR pointing to the MET on derecho.  Pointing this to the correct MET on casper fixes the issue.  Sorry about that.

Tina

**Reply by jprestop** [reply]
Ah!  Great!  Thank you for letting me know.  I'm glad you found the problem and were able to resolve it.
